components/Workouts/WorkoutCard/index.jsx

```diff
@@ -0,0 +1,64 @@
+import Heading from "@/components/Atoms/Text/Heading";
+import HeadingTiny from "@/components/Atoms/Text/HeadingTiny";
+import styled from "styled-components";
+
+export default function WorkoutCard({ workout, exercises }) {
+  const usedMuscle = [];
+  exercises.map((exercise) => usedMuscle.push(...exercise.muscleGroups));
+  const uniqueMuscles = [...new Set(usedMuscle)];
+
+  const muscleCount = {};
+  for (const muscle of usedMuscle) {
+    muscleCount[muscle] = muscleCount[muscle] ? muscleCount[muscle] + 1 : 1;
+  }
+
+  return (
+    <StyledWorkoutCard>
+      <h3>{workout.name}</h3>
+      <>
+        <Heading>Exercises</Heading>
+        {exercises.map((exercise, index) => {
+          const exerciseDataInWorkout = workout.exercises.find(
+            (workoutExercise) => workoutExercise.exerciseId === exercise._id
+          );
+          return (
+            <StyledParagraph key={index}>
+              {`${exercise.name} ---
+              reps:_ ${exerciseDataInWorkout.reps}
+              sets:_ ${exerciseDataInWorkout.sets}
+              `}
+            </StyledParagraph>
+          );
+        })}
+      </>
+      <Heading>Muscle Groups</Heading>
+      <StyledList>
+        {uniqueMuscles.map((muscle, index) => {
+          return <HeadingTiny key={index}>{muscle}</HeadingTiny>;
```

Comment:
```suggestion
          return <HeadingTiny key={muscle}>{muscle}</HeadingTiny>;
```
Use the `muscle` as a key here, as it's already unique.